### ISSUE TITLE
Error on https://www.newgrounds.com/portal/view/648589

### ISSUE THREAD

**Issue by p0008874** [ruffle-rs/ruffle#8558]
# Error Info
Error name: ChunkLoadError
Error message: Loading chunk 966 failed.
(error: https://uploads.ungrounded.net/ruffle_wrapper/lib/966.js)
Error stack:
```
ChunkLoadError: Loading chunk 966 failed.
(error: https://uploads.ungrounded.net/ruffle_wrapper/lib/966.js)
    at s.f.j (chrome-extension://donbcfbmhbcapadipfkeojnmajbakjdc/dist/ruffle.js?id=85555344165:1:2068)
    at chrome-extension://donbcfbmhbcapadipfkeojnmajbakjdc/dist/ruffle.js?id=85555344165:1:467
    at Array.reduce (<anonymous>)
    at s.e (chrome-extension://donbcfbmhbcapadipfkeojnmajbakjdc/dist/ruffle.js?id=85555344165:1:445)
    at chrome-extension://donbcfbmhbcapadipfkeojnmajbakjdc/dist/ruffle.js?id=85555344165:1:5542
    at async E.ensureFreshInstance (chrome-extension://donbcfbmhbcapadipfkeojnmajbakjdc/dist/ruffle.js?id=85555344165:1:23792)
    at async E.load (chrome-extension://donbcfbmhbcapadipfkeojnmajbakjdc/dist/ruffle.js?id=85555344165:1:26657)
```

# Player Info
Allows script access: true

# Page Info
Page URL: https://uploads.ungrounded.net/ruffle_wrapper/ruffleembed.html?v=1.0.67&useloader=0&browsermode=default&props=%7B%22ruffle%22%3A%22%2Fruffle_wrapper%2Flib%2Fruffle.js%3F1668524747%22%2C%22public_path%22%3A%22%2Fruffle_wrapper%2Flib%2F%22%2C%22swf%22%3A%22https%3A%2F%2Fuploads.ungrounded.net%2F648000%2F648589_mord-019.swf%3F1415891741%22%2C%22vars%22%3A%7B%22ngio_session_id%22%3A%2227175267.0d8ef9228f473671479b9f5c6ee04f3ca34d9d63435784%22%2C%22NewgroundsAPI_PublisherID%22%3A1%2C%22NewgroundsAPI_SandboxID%22%3A%226374fdddc052c%22%2C%22NewgroundsAPI_SessionID%22%3A%22Fw7mKTbkZK6nOxWFGKu634d94adde261d93d156c4ff0db26e35c15741bfbeTF6%22%2C%22NewgroundsAPI_UserName%22%3A%22p0008874%22%2C%22NewgroundsAPI_UserID%22%3A10854601%2C%22ng_username%22%3A%22p0008874%22%7D%2C%22width%22%3A640%2C%22height%22%3A360%2C%22icon%22%3A%22https%3A%2F%2Fpicon.ngfiles.com%2F648000%2Fflash_648589_card.png%3Ff1606138639%22%2C%22warnOnUnsupportedContent%22%3Afalse%7D

# Browser Info
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36
Platform: Win32
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2022-11-16
Channel: nightly
Built: 2022-11-16T00:18:41.332Z
Commit: b3debc96afab2882c70ee27cf4dd5230874e803b
Is extension: false

# Metadata

**Comment by n0samu** [comment]
Hi, can you provide the game page URL? Regardless, this looks like the Ruffle extension interfering with Newgrounds's selfhosted copy of Ruffle, which is a known problem.

**Comment by p0008874** [comment]
> Hi, can you provide the game page URL? Regardless, this looks like the Ruffle extension interfering with Newgrounds's selfhosted copy of Ruffle, which is a known problem.


https://user-images.githubusercontent.com/75534590/202411410-71ef93b7-65d2-4165-a4ad-582fe747195d.mp4

**Comment by n0samu** [comment]
Fixed by #11678
